k8s-install/canal_etcd_tls.yaml

```diff
@@ -422,22 +422,22 @@ spec:
 apiVersion: rbac.authorization.k8s.io/v1beta1
 kind: ClusterRoleBinding
 metadata:
-  name: calico-cni-plugin
+  name: canal
```

Comment:
Nah, there's only one binding for etcd mode, and it binds the canal serviceaccount to the canal role.

The `canal-calico` etc was because there were multiple bindings.